supertrader/data/binance.py

```
@@ -0,0 +1,17 @@
+import pandas as pd
+from binance import Client
+
+class HistoricalData:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)